nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/features/PlatformFunctions.java

```diff
@@ -263,4 +265,11 @@ public boolean documentExists(CoreSession session, String idOrPath) {
         return session.exists(documentRef);
     }
 
+    /**
+     * @since 11.1
+     */
+    public DateWrapper calendar(String test) {
+        return new DateWrapper(new Date());
```

Comment:
What's the purpose of this method?

Reply:
https://jira.nuxeo.com/browse/NXP-26847?focusedCommentId=379504&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-379504

Reply:
AFAIK the ticket explains that we are missing a method `PlatformFunctions#calendar` that takes a `Date` as a parameter and returns a `DateWrapper`, like the `CoreFunctions#calendar` method.

Here we have a method that takes a useless `String` as the only parameter, and returns a `DateWrapper` always wrapping `now`.

The following snippet (using the new method) does not seem correct to me :)
```java
DateWrapper dw = new PlatformFunctions().calendar("foo");
...
```

You should add a Java test for that method on the `PlatformFunctionTest` class besides the Scripting test.

Reply:
Yes I didn't pay attention, not it's fixed